tee-worker/ts-tests/resuming_worker.test.ts

```diff
@@ -1,151 +1,290 @@
-                    nodeUrl +
-                    ' --node-port ' +
-                    nodePort +
-                    ' run --skip-ra',
-            },
+const workerConfig: Record<'worker0' | 'worker1', WorkerConfig> = {
+    worker0: {
+        name: 'worker0',
+        muRaPort: 3443,
+        untrustedHttpPort: 4545,
+        trustedWorkerPort: 2000,
+        untrustedWorkerPort: 3000,
+        enableMockServer: true,
+        requestStateOnLaunch: false,
+    },
+    worker1: {
+        name: 'worker1',
+        muRaPort: 3444,
+        untrustedHttpPort: 4546,
+        trustedWorkerPort: 2001,
+        untrustedWorkerPort: 3001,
+        enableMockServer: false,
+        requestStateOnLaunch: true,
+    },
+} as const;
+
+type NodeConfig = {
+    nodeUrl: string;
+    nodePort: number;
+};
+
+type Command = 'launch' | 'resume';
+
+function generateWorkerCommandArguments(
+    command: Command,
+    { nodeUrl, nodePort }: NodeConfig,
+    workerParams: WorkerConfig
+): string {
+    const isLaunch = command === 'launch';
+
+    return [
+        '--running-mode mock',
+        ...(workerParams.enableMockServer ? ['--enable-mock-server'] : []),
+        ...(isLaunch ? ['--clean-reset'] : []),
+        '--mu-ra-external-address localhost',
+        `--mu-ra-port ${workerParams.muRaPort}`,
+        `--untrusted-http-port ${workerParams.untrustedHttpPort}`,
+        '--ws-external',
+        '--trusted-external-address wss://localhost',
+        `--trusted-worker-port ${workerParams.trustedWorkerPort}`,
+        '--untrusted-external-address ws://localhost',
+        `--untrusted-worker-port ${workerParams.untrustedWorkerPort}`,
+        `--node-url ${nodeUrl}`,
+        `--node-port ${nodePort}`,
+        `run`,
+        `--skip-ra`,
+        ...(isLaunch && workerParams.requestStateOnLaunch ? ['--request-state'] : []),
+        ...(isLaunch ? ['--dev'] : []),
+    ].join(' ');
+}
+
+function initializeFiles(workingDir: string, binaryDir: string) {
+    fs.mkdirSync(workingDir, { recursive: true });
+    fs.copyFileSync(path.join(binaryDir, 'enclave.signed.so'), path.join(workingDir, 'enclave.signed.so'));
+    fs.copyFileSync(path.join(binaryDir, 'integritee-service'), path.join(workingDir, 'integritee-service'));
+    fs.closeSync(fs.openSync(path.join(workingDir, 'spid.txt'), 'w'));
+    fs.closeSync(fs.openSync(path.join(workingDir, 'key.txt'), 'w'));
+    const data = JSON.stringify(
+        {
+            twitter_official_url: 'http://localhost:19527',
+            twitter_litentry_url: 'http://localhost:19527',
+            twitter_auth_token: '',
+            discord_official_url: 'http://localhost:19527',
+            discord_litentry_url: 'http://localhost:19527',
+            discord_auth_token: '',
+            achainable_url: 'http://localhost:19527',
+            achainable_auth_key: '',
         },
-    };
+        null,
+        4
+    );
+    fs.writeFileSync(`${workingDir}/worker-config-mock.json`, data);
```

Comment:
I don't see this file mentioned in the worker command-line arguments in the function `generateWorkerCommandArguments` , Are you sure we need this file? I might be missing something here.

Reply:
I don't know; it was there before so I didn't want to mess with it :D

My guess would be that the worker picks it up when `--enable-mock-server` is passed, but I can't find an explicit mention about it in the code.

Reply:
(although I did forget to switch to `path.join` for this one, so thanks for pointing it out :grin:)

Reply:
I don't think it picks it up, Because I use `--enable-mock-server` and it doesn't give me an error

Reply:
Well, It's a trivial operation, I just wanted to check if I have missed this in my restart script 🤣

Reply:
It worked :raised_hands: 
Good catch @felixfaisal; thanks!